test/verify/check-machines

```diff
@@ -202,6 +204,9 @@ class TestMachines(MachineCase):
         b.wait_present("{0}-state".format(self.providerVm1_id))
         b.wait_in_text("{0}-state".format(self.providerVm1_id), "running")
 
+        b.wait_present("#test-vm-action-{0}".format(self.providerVm1_name)) # is provider-specific action button rendered?
+        b.wait_in_text("#test-vm-props-{0}".format(self.providerVm1_name), "Test Provider Property") # are provider-specific VM props rendered?
+
```

Comment:
@dperpeet , please have a look.
Unless reading the cpu type/emulated machine from the machine and comparing it back to UI (which is probably not needed), I'm unable to check for appropriate value.
I believe this is still sufficient:
- cpu type is either "host" (as expected from virt-install params) or "custom" (for debian-8)
- emulated machine value is asserted to be non-empty (depends on test machine)